modules/custom/activity_viewer/src/Plugin/views/filter/ActivityNotificationVisibilityAccess.php

```diff
@@ -231,6 +231,11 @@ public function query() {
       $or->condition($membership_access);
     }
 
+    // Simple condition that allows to receive the notifications sent directly
+    // to the recipient.
+    $or
```

Comment:
@nechai 
- Add defensive check to ensure that user is authenticated.
- I think we can add changes in one line.

Reply:
@Ressinel Fixed